src/reducers/index.js

```diff
@@ -9,8 +9,10 @@ export default (count = 0, action) => {
   switch (action.type) {
     case INCREMENT:
     // Fill in the body of this case
+      return count + 1;
```

Comment:
`++count` and `--count` are great shorthand ways to increment or decrement a variable. Take a look at the difference however between `++count` vs. `count++`.